StrangeCalculatorTask/src/main/java/strangecalculator/Menu.java

```diff
@@ -0,0 +1,51 @@
+package strangecalculator;
+
+import strangecalculator.calculator.CalculatorInterface;
+import strangecalculator.calculator.GoogleCalculator;
+import strangecalculator.calculator.LocalCalculator;
+import strangecalculator.reader.ConsoleReader;
+import strangecalculator.reader.ReaderInterface;
+import strangecalculator.utils.Terminal;
+
+import java.io.IOException;
+
+public class Menu {
+    public CalculatorInterface getCalculatorType() throws IOException, InterruptedException {
+        CalculatorInterface calculatorType = null;
+        Terminal.print("Select an option that you want and press ENTER: 1) - Local calculator 2) - Google Calculator 3) Exit ");
```

Comment:
You could add a line break to the string using "\n", to print text to few lines, so you will have not:
```select option: 1) first 2) second```

but: 
```select option:
1) first
2) second```